lib/isolator/adapters/background_jobs/active_job.rb

```diff
@@ -7,4 +7,9 @@
   details_message: ->(obj) {
     "#{obj.class.name}" \
     "#{obj.arguments.any? ? " (#{obj.arguments.join(", ")})" : ""}"
-  }
+  },
+  disabled: begin
+    config = ActiveJob::Base.try(:enqueue_after_transaction_commit)
```

Comment:
I think, we should take into the current class, not the base class: https://github.com/rails/rails/pull/51426/files#diff-0f10b9da3d2494e6fd1edf3e00000397f5c3c3cc26a76f4a36a959d3f236be4eR20

And I think, there will be a common situation when some legacy jobs (or third-party jobs from gems) will still use legacy interface.

Given that, I'd suggest considering the `#ignore_if` feature for this workaround.

Reply:
> I think, we should take into the current class, not the base class: [rails/rails#51426 (files)](https://github.com/rails/rails/pull/51426/files#diff-0f10b9da3d2494e6fd1edf3e00000397f5c3c3cc26a76f4a36a959d3f236be4eR20)

Ah, nice catch!

> Given that, I'd suggest considering the #ignore_if feature for this workaround.

Sounds good, I'll address when I get a chance.

Reply:
Actioned in [these changes](https://github.com/palkan/isolator/compare/5ddaac8b5a71e329ba283c2c40712a81d4ae107d..d50ea6217048eb091676ae1fec6db71e6dcd08cb) (although, it might be easier to review the final diff).